File: Java/src/com/vaishakh/Algorithms/linearsearch.java
package com.vaishakh.Algorithms;

import java.util.Scanner;

public class linearsearch {
    public static void main(String[] args) {
        int flag=0,i;
        Scanner in = new Scanner(System.in);
        System.out.println("Enter the size of the array");
        int n = in.nextInt();
        int[] arr = new int[n];
        System.out.println("Enter the elements of the array");
        for (i = 0; i < n; i++) {
            arr[i] = in.nextInt();
        }
        System.out.println("Enter the element to be searched for in the array");
        int x = in.nextInt();
        for (i = 0; i < n; i++) {
            if(arr[i] == x){
                flag=1;
            }
        }
        if(flag==1){
            System.out.println("Element "+x+" is present in the array");
        }
        else{
            System.out.println("Element "+x+" is not present in the array");
        }
    }
}

File: Python/Basic/linearsearch.py
def search(x,lst1):
    n=len(lst1)
    flag=0
    for i in range(0,n):
        if x==lst1[i]:
            flag=1
    if flag==1:
        print("The element",x,"is present in the list")
    if flag==0:
        print("The element",x,"is not present in the list")

l=int(input("Enter the no of elements in the array"))
arr=[]
print("Enter the elements")
for i in range(0,l):
    y=int(input())
    arr.append(y)
n=int(input("enter the element to be searched for"))
search(n,arr)